gvsbuild/projects/protobuf.py

```diff
@@ -26,8 +26,9 @@ def __init__(self):
         Project.__init__(
             self,
             "protobuf",
-            version="3.21.12",
-            archive_url="https://github.com/protocolbuffers/protobuf/releases/download/v{minor}.{micro}/protobuf-cpp-{version}.tar.gz",
+            # Overall project version for lastversion detection
+            version="21.12",
+            archive_url="https://github.com/protocolbuffers/protobuf/releases/download/v3.{major}.{minor}/protobuf-cpp-3.{version}.tar.gz",
```

Comment:
`lastversion_major=3`?

No need to change the version (and we want version to be accurate).

Reply:
Nice, I didn't know about that feature! I think we want it set to 3.